compiler/data/vertex-adaptor.h

```diff
@@ -7,170 +7,159 @@
 #include "auto/compiler/vertex/is-base-of.h"
 #include "common/wrappers/fmt_format.h"
 
+#include "compiler/location.h"
 #include "compiler/operation.h"
-#include "compiler/stage.h"
 
 template<Operation Op>
 class vertex_inner;
 
 template<Operation Op>
 class VertexAdaptor;
 
-class Location;
-
 using VertexPtr = VertexAdaptor<meta_op_base>;
 
 VertexPtr clone_vertex(VertexPtr);
 
 template<Operation Op>
 class VertexAdaptor {
-  vertex_inner<Op> *impl;
+  vertex_inner<Op> *impl_{nullptr};
```

Comment:
If you have renamed this field, please update lldb_addons.py to remain pretty printers work

Reply:
done